doc/pfc_asym/PFC_Asymmetric_Test_HLD.md

```diff
@@ -428,3 +429,27 @@ setup, pfc_storm_runner, enable_pfc_asym
 
 - Teardown:
   - Stop ARP responder
+
+### Test case # 5 - Priority Group dropped packets testing
+#### Test objective
+Asymmetric PFC is disabled. Verify ingress Priority Group drop packets counter increasing
+
+#### Used fixtures
+setup, pfc_storm_runner
+
+#### Test steps
+- Setup:
+  - Start ARP responder
```

Comment:
Mention why ARP responder is needed, how many servers should ARP responder simulate?
Could it be better to just "ip neigh add ... nud persistent"?